### DISCUSSION TITLE
Red/Comms

### DISCUSSION THREAD

**Discussion by greggirwin** [red/red#5282] [General]
Red Comm Channel discussion

**Comment by greggirwin** [comment]
The Gitter/Matrix changeover is coming soon. If there are problems or questions, people can reach out here. We can also talk about alternative comm systems for future use, if needed.

**Comment by rebolek** [comment]
Archive of all conversation from Gitter:

https://mega.nz/file/U1AAGCiJ#0Aymh7v-oByammtpx2x4O7oma2HKGYwxan05W-U5o9s
